dashboard/package.json

```diff
@@ -39,7 +39,8 @@
     "stylus": "^0.51.1",
     "underscore": "^1.8.3",
     "util": "^0.10.3",
-    "js-yaml": "^2.0.5"
+    "js-yaml": "^2.0.5",
+    "sinon": "^1.15.3"
```

Comment:
sinon should be added as "devDependency" (next time run `npm install xxxx --save-dev`)